src/components/NavBar/NavBar.css

```diff
@@ -10,7 +10,48 @@ nav {
 .logo-heading {
   font-family: 'Sacramento', cursive;
   font-size: xxx-large;
-  margin: 1% 7% 1% 1%;
+  margin: 1% 1% 1% 1%;
   flex: 1;
   text-align: center; 
+}
+
+.login-container {
+  display: flex;
+  align-items: center;
+  height: 60px;
+  word-wrap: break-word;
+}
+
+a {
+  text-decoration: none;
+}
+
+a:visited {
+  text-decoration: none;
+  color: black;
+}
```

Comment:
should we make it black all the time, not just when visited since the option to log in is blue otherwise?

Reply:
oops i didn't fix this but yes lets one of us get this in our next branch